common/buildcraft/transport/tile/TilePipeHolder.java

```diff
@@ -72,6 +72,9 @@
     public static final int NET_UPDATE_PLUG_EAST = getReceiverId(PipeMessageReceiver.PLUGGABLE_EAST);
     public static final int NET_UPDATE_WIRES = getReceiverId(PipeMessageReceiver.WIRES);
 
+    private int[] redstoneValues = new int[6];
+    private int[] oldredstoneValues = new int[]{ -1, -1, -1, -1, -1, -1};
```

Comment:
in case of bad nbt it won't matter realy if i assign or copy over, it's pretty much impossible to protect against nbt being corrupted or tampered with